src/components/MediaUpload.tsx

```diff
@@ -272,15 +288,26 @@ function MediaUpload(props: IProps) {
         <DialogTitle id="audUploadDlg">{title[uploadType]}</DialogTitle>
         <DialogContent>
           <DialogContentText>{text[uploadType]}</DialogContentText>
-          <div className={classes.drop}>
-            <DropTarget
-              name={name}
-              handleFiles={handleFiles}
-              acceptextension={acceptextension}
-              acceptmime={acceptmime}
-              multiple={multiple}
-              t={t}
+          {uploadType !== UploadType.IntellectualProperty && (
```

Comment:
This seems like it will show up too often.  Do we need the rights for resource uploads (maybe?)   But definitely not ITF/PTF

Reply:
For now, let's just show when we're uploading media.